### PR TITLE
Fix failure brought by block dev change

### PR DESCRIPTION
**Pull request by chunfuwen**
that 'host_device' driver expects either a character or block device

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
By the way, fix one error in python code check with escape character

**Comment by chunfuwen** [comment]
'''paste the result here"
Command 'virsh list' finished with 0 after 0.21374893188476562s
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.multidisks.coldplug.multi_disks_test.virtio_disks_order

File: libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py [hunk 1]
```diff
@@ -1032,7 +1035,7 @@ def check_info_in_libvird_log_file(matchedMsg=None):
     if test_special_characters_xml:
         first_disk = vm.get_first_disk_devices()
         first_disk_source = first_disk['source']
-        first_disk_source_rename = "%s.rhel7.4\.img**" % first_disk_source
+        first_disk_source_rename = "%s.rhel7.4\\.img**" % first_disk_source
```
Comment (Yingshun) [inline libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py]: why do you need extra '\' here?

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py]: it was a warning from python3 inspekt if without double \

Comment (Yingshun) [reply libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py]: no warning if I only add L807-809.

Python 3.6.6
(p3-ve) [ycui@host193 virtual_disks]$ inspekt checkall --disable-style E501,E265,W601,W605,E402,E722,E741 --no-license-check virtual_disks_multidisks.py 
PEP8 disabled: E501,E265,W601,W605,E402,E722,E741
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011
Pylint enabled : W0611
License check: disabled
(p3-ve) [ycui@host193 virtual_disks]$ 

Are you quite sure double \ is required?

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py]: (p3-ve) [root@chunfulap2 tp-libvirt]# inspekt style libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py
PEP8 disabled: E501,E265,W601,E402
libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py:1035:47: W605 invalid escape sequence '\.'
Python library autopep8 not installed. Please install it if you want to use --fix
Style: libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py FAIL (FIX NOT OK)
PEP8 compliance FAIL

By using double / , w605 issue can be fixed in this case

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py]: remove this change to another PR to avoid confusion.

**Comment by chunfuwen** [comment]
paste new test log:
'''
[stdout] systemd
legacy start libvirtd daemon IGNORED with function name: restart
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.multidisks.coldplug.multi_disks_test.virtio_disks_order
'''

**Pull request merged**